core/src/main/java/com/hcl/ecomm/core/services/impl/MySqlServiceImpl.java

```diff
@@ -79,16 +87,19 @@ public void userComplaintSubmission(String first_Name, String last_Name, String
 				LOG.error("Error while trying to close connection. SQLException = {}", e);
 			}
 		}
-		
 		LOG.debug("userComplaintSubmission method end.");
+		return complaintId;
 	}
 
-	private void triggerNotifyMail(String first_Name, String email, String template) {
+	private void triggerNotifyMail(String first_Name, String email, String template, String complaintId) {
 		Map emailParams = new HashMap<>();
 		LOG.debug("triggerNotifyMail() methid start: Email="+email);
 		String templatePath = StringUtils.EMPTY;
 		if(template.equals("closingComplaint")) {
 			 templatePath = "/etc/notification/email/hclecomm/user-complaint-resolved-email-template.html";	
+			 emailParams.put("positveFeedbackUrl", "http://localhost:4502/content/hclecomm/us/en/complaint-feedback.html?wcmmode=disabled&satisfied=yes&complaintId="+complaintId);
+			 emailParams.put("negativeFeedbackUrl", "http://localhost:4502/content/hclecomm/us/en/complaint-feedback.html?wcmmode=disabled&satisfied=no&complaintId="+complaintId);
```

Comment:
@rakeshkumar-verma : why do we have absolute URL's here ? "http://localhost:4502/"

Reply:
@rishabh-behl  updated the code